docs/workflows/create_workflow_block.md

```diff
@@ -0,0 +1,1898 @@
+    from typing import Literal, Union, List, Optional, Type
+    from pydantic import Field
+    from inference.core.workflows.prototypes.block import (
+        WorkflowBlockManifest,
+        WorkflowBlock,
+        BlockResult,
+    )
+    from inference.core.workflows.execution_engine.entities.base import (
+        OutputDefinition,
+        WorkflowImageData,
+    )
+    from inference.core.workflows.execution_engine.entities.types import (
+        StepOutputImageSelector,
+        WorkflowImageSelector,
+        FloatZeroToOne,
+        WorkflowParameterSelector,
+        FLOAT_ZERO_TO_ONE_KIND,
+        BOOLEAN_KIND,
+    )
+    
+    class ImagesSimilarityManifest(WorkflowBlockManifest):
+        type: Literal["my_plugin/images_similarity@v1"] 
+        name: str
+        image_1: Union[WorkflowImageSelector, StepOutputImageSelector] = Field(
+            description="First image to calculate similarity",
+        )
+        image_2: Union[WorkflowImageSelector, StepOutputImageSelector] = Field(
+            description="Second image to calculate similarity",
+        )
+        similarity_threshold: Union[
+            FloatZeroToOne,
+            WorkflowParameterSelector(kind=[FLOAT_ZERO_TO_ONE_KIND]),
+        ] = Field(
+            default=0.4,
+            description="Threshold to assume that images are similar",
+        )
+        
+        @classmethod
+        def describe_outputs(cls) -> List[OutputDefinition]:
+            return [
+              OutputDefinition(
+                name="images_match", 
+                kind=[BOOLEAN_KIND],
+              ),
+            ]
+    
+        @classmethod
+        def get_execution_engine_compatibility(cls) -> Optional[str]:
+            return ">=1.0.0,<2.0.0"
+    
+        
+    class ImagesSimilarityBlock(WorkflowBlock):
+      
+        @classmethod
+        def get_manifest(cls) -> Type[WorkflowBlockManifest]:
+            return ImagesSimilarityManifest
+    
+        def run(
+            self,
+            image_1: WorkflowImageData,
+            image_2: WorkflowImageData,
+            similarity_threshold: float,
+        ) -> BlockResult:
+            pass
+    ```
+
+    * lines `1`, `5-6` and `8-9` added changes into import surtucture to 
+    provide additional symbols required to properly define block class and all
+    of its methods signatures
+
+    * line `59` defines class method `get_manifest(...)` to simply return 
+    the manifest class we cretaed earlier
+
+    * lines `62-68` define `run(...)` function, which Execution Engine
+    will invoke with data to get desired results
+
+### Providing implementation for block logic
+
+Let's now add example implementation of `run(...)` method to our block, such that
+it can produce meaningful results.
+
+!!! Note
+    
+    Content of this section is supposed to provide example on how to interact 
+    with Workflow ecosystem as block creator, rather than providing robust 
+    implementation of the block.
```

Comment:
```suggestion
    The Content of this section is supposed to provide examples on how to interact 
    with the Workflow ecosystem as block creator, rather than providing robust 
    implementation of the block.
```

Reply:
👍